src/BUILD

```diff
@@ -134,6 +134,7 @@ cc_shared_library(
     features = [],
     roots = [
         "//src:ovms_lib",
+        "@llm_engine//:llm_engine", "@libgit2_engine//:libgit2_engine",
```

Comment:
Why is it needed here now and was not needed before?

Reply:
With FUZZER_BUILD=1 we only build this target. The shared bazel library will not build nor add this dependencies so it will not be available in bazel-out directory.